### PR TITLE
Implements separate asset issuer account #129

### PR DESCRIPTION
**Pull request by ronaldoy**

### CODE REVIEW

**Comment by msfeldstein** [comment]
This lgtm but @debnil can you take a look to make sure i didn't miss something django specific?

File: src/app/settings.py [hunk 1]
```diff
@@ -39,6 +39,7 @@
 
 STELLAR_ACCOUNT_ADDRESS = env("STELLAR_ACCOUNT_ADDRESS")
 STELLAR_ACCOUNT_SEED = env("STELLAR_ACCOUNT_SEED")
+STELLAR_ASSET_ISSUER = env("STELLAR_ACCOUNT_ADDRESS")
```
Comment (debnil) [inline src/app/settings.py]: This should be `env("STELLAR_ASSET_ISSUER")`

File: src/transaction/management/commands/watch_transactions.py [hunk 2]
```diff
@@ -27,7 +27,7 @@ def _check_payment_op(operation, want_asset, want_amount):
     if str(operation.asset.code) != want_asset:
         return False
     # TODO: Handle assets not issued by the anchor address.
```
Comment (debnil) [inline src/transaction/management/commands/watch_transactions.py]: This TODO should either be removed or changed. Maybe to ```# TODO: Handle multiple possible asset issuance accounts.```?

Comment (ronaldoy) [reply src/transaction/management/commands/watch_transactions.py]: Done!

File: src/app/settings.py (hunk 1 above)
Comment (ronaldoy) [reply src/app/settings.py]: Thank you for catching this!

**Pull request merged**